src/main/java/org/toradocu/extractor/JavadocExtractor.java

```diff
@@ -61,7 +61,9 @@ public DocumentedType extract(String className, String sourcePath)
     List<String> classesInPackage = new ArrayList<String>();
     for (File file : listOfFiles) {
       String name = parseClassName(file.getName(), className);
-      if (!name.equals(className) && !name.contains("package-info")) classesInPackage.add(name);
+      if (name != null && !name.equals(className) && !name.contains("package-info")) {
```

Comment:
I introduced these instructions because otherwise some Exception classes couldn't be found, so I honestly didn't think about the Javadoc: if we think this is useful in general, we can keep it